python_modules/dagster/dagster_tests/cli_tests/command_tests/test_materialize_command.py

```diff
@@ -177,3 +177,16 @@ def test_partition_range_multi_run_backfill_policy():
 def test_failure():
     result = invoke_materialize("fail_asset")
     assert result.exit_code == 1
+
+
+def test_run__cli_config_json():
+    with instance_for_test() as instance:
+        runner = CliRunner()
+        options = ["-f", file_relative_path(__file__, "assets.py"), "--select", "asset_with_config", "--config-json", "{\"some_prop\": \"foo\"}"]
+
+        result = runner.invoke(asset_materialize_command, options)
+
+        
+        #assert "some_prop:foo" in result.output
+        #assert instance.get_latest_materialization_event(AssetKey("asset_with_config")) is not None
```

Comment:
the thing that's important to test here is that the config actually made its way into the asset (this can be accomplished by asserting the config value within the body of `asset_with_config`:

```python
@asset
def asset_with_config(context: AssetExecutionContext, config: MyConfig):
    assert config.some_prop == "foo"
    context.log.info(f"some_prop:{config.some_prop}"
```

Reply:
Keeping the chain of PRs alive, this should be solved in:

https://github.com/jimmyshah/dagster/pull/1/commits/1672182a13c2d4a2325720b2074a9ef5ba36e609